src/components/sidebar/sidebar.component.tsx

```diff
@@ -88,7 +95,14 @@ const Sidebar: React.FC<SidebarProps> = ({
             })}
             key={index}
             onClick={() => handleClick(page.label)}>
-            <div className={styles.sectionLink}>{page.label}</div>
+            <div className={styles.sectionLink}>
+              {page.label}
+              {isCollapsed ? (
+                <ChevronUpIcon className={styles.collapseIcon} aria-label="Expand" />
+              ) : (
+                <ChevronDownIcon className={styles.collapseIcon} aria-label="Collapse" />
+              )}
```

Comment:
Am having a little bit of an issue with this, The sidebar only displays the page title for quick navigation. Do u have a screenshot of what your trying to achieve here

Reply:
Yes, there was no UI element to show the pages are collapsible like the sections so i added the ChevronUp and Down.
Screenshot
<img width="1436" alt="Screenshot 2024-09-19 at 13 47 01" src="https://github.com/user-attachments/assets/5274f3e0-464f-47d0-be28-41a50e729ace">

Reply:
See previous comment as it addresses this as well

Reply:
This is a previous design with the sidenav visible https://app.zeplin.io/project/60d5947dd636aebbd63dce4c/screen/61bb008f0e69e45e7175e63e